core/src/main/java/tech/pegasys/web3signer/core/service/http/handlers/signing/eth2/Eth2SigningRequestBody.java

```diff
@@ -20,135 +20,24 @@
-  public ArtifactType getType() {
-    return type;
-  }
-
-  @JsonProperty("fork_info")
-  public ForkInfo getForkInfo() {
-    return forkInfo;
-  }
-
-  @JsonProperty("block")
-  public BeaconBlock getBlock() {
-    return beaconBlock;
-  }
-
-  @JsonProperty("beacon_block")
-  public BlockRequest getBlockRequest() {
-    return blockRequest;
-  }
-
-  @JsonProperty("attestation")
-  public AttestationData getAttestation() {
-    return attestation;
-  }
-
-  @JsonProperty("signing_root")
-  @JsonAlias("signingRoot")
-  public Bytes getSigningRoot() {
-    return signingRoot;
-  }
-
-  @JsonProperty("aggregation_slot")
-  public AggregationSlot getAggregationSlot() {
-    return aggregationSlot;
-  }
-
-  @JsonProperty("aggregate_and_proof")
-  public AggregateAndProof getAggregateAndProof() {
-    return aggregateAndProof;
-  }
-
-  @JsonProperty("voluntary_exit")
-  public VoluntaryExit getVoluntaryExit() {
-    return voluntaryExit;
-  }
-
-  @JsonProperty("randao_reveal")
-  public RandaoReveal getRandaoReveal() {
-    return randaoReveal;
-  }
-
-  @JsonProperty("deposit")
-  public DepositMessage getDeposit() {
-    return deposit;
-  }
-
-  @JsonProperty("sync_committee_message")
-  public SyncCommitteeMessage getSyncCommitteeMessage() {
-    return syncCommitteeMessage;
-  }
-
-  @JsonProperty("sync_aggregator_selection_data")
-  public SyncAggregatorSelectionData getSyncAggregatorSelectionData() {
-    return syncAggregatorSelectionData;
-  }
-
-  @JsonProperty("contribution_and_proof")
-  public ContributionAndProof getContributionAndProof() {
-    return contributionAndProof;
-  }
-
-  @JsonProperty("validator_registration")
-  public ValidatorRegistration getValidatorRegistration() {
-    return validatorRegistration;
-  }
-}
+public record Eth2SigningRequestBody(
+    @JsonProperty(value = "type", required = true) ArtifactType type,
+    @JsonProperty("signing_root") @JsonAlias("signingRoot") Bytes signingRoot,
+    @JsonProperty("fork_info") ForkInfo forkInfo,
+    @JsonProperty("block") BeaconBlock block,
+    @JsonProperty("beacon_block") BlockRequest blockRequest,
+    @JsonProperty("attestation") AttestationData attestation,
+    @JsonProperty("aggregation_slot") AggregationSlot aggregationSlot,
+    @JsonProperty("aggregate_and_proof") AggregateAndProof aggregateAndProof,
+    @JsonProperty("voluntary_exit") VoluntaryExit voluntaryExit,
+    @JsonProperty("randao_reveal") RandaoReveal randaoReveal,
+    @JsonProperty("deposit") DepositMessage deposit,
+    @JsonProperty("sync_committee_message") SyncCommitteeMessage syncCommitteeMessage,
+    @JsonProperty("sync_aggregator_selection_data")
+        SyncAggregatorSelectionData syncAggregatorSelectionData,
+    @JsonProperty("contribution_and_proof") ContributionAndProof contributionAndProof,
+    @JsonProperty("validator_registration") ValidatorRegistration validatorRegistration,
+    @JsonProperty("blob_sidecar") BlindedBlobSidecar blindedBlobSidecar) {}
```

Comment:
blob_sidecar or blindedBlobSidecar?

Reply:
`@JsonProperty("blob_sidecar")` is intended. However, I'll be changing `BlindedBlobSidecar` to `BlobSidecar`.